.github/workflows/build-mudlet.yml

```diff
@@ -157,6 +172,54 @@ jobs:
       working-directory: '${{runner.workspace}}/b/ninja'
       run: ctest --output-on-failure
 
+    - name: (Linux/macOS) restore Luarocks for packaging
+      uses: actions/cache@v2
+      with:
+        path: $HOME/.luarocks
+        key: ccache-${{matrix.os}}-${{matrix.compiler}}-${{hashFiles('.github/workflows/build-mudlet.yml')}}
+        restore-keys: ccache-${{matrix.os}}-${{matrix.compiler}}-
+      if: runner.os == 'Linux' || runner.os == 'macOS'
+
+    - name: (Linux) package Mudlet
+      if: runner.os == 'Linux' && matrix.deploy == 'deploy'
+      run: |
+        export PATH="${{env.VCPKG_ROOT}}/installed/x64-linux/tools/lua:$PATH"
+
+        luarocks install --local LuaFileSystem
+        luarocks install --local luautf8
+        luarocks install --local lua-zip
+        luarocks install --local luautf8
+        luarocks install --local LuaSQL-SQLite3 SQLITE_INCDIR="${{env.VCPKG_ROOT}}/installed/x64-linux/include"
```

Comment:
Are you sure it is `LuaSQL-SQLite3` not `luaSQL-SQLite3` - assuming that it *is* case-sensitive?

Reply:
Yeah, it is the same as we had in Travis before.